pkg/operator/operatorclient.go

```diff
@@ -0,0 +1,36 @@
+package operator
+
+import (
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+type OperatorClient struct {
+}
+
+var _ v1helpers.OperatorClient = &OperatorClient{}
+
+func (c OperatorClient) GetObjectMeta() (*metav1.ObjectMeta, error) {
+	return nil, fmt.Errorf("not implemented")
+}
+
+func (c OperatorClient) Informer() cache.SharedIndexInformer {
+	return nil
+}
+
+func (c OperatorClient) GetOperatorState() (*operatorv1.OperatorSpec, *operatorv1.OperatorStatus, string, error) {
+	return nil, nil, "", fmt.Errorf("not implemented")
+}
+
+func (c OperatorClient) UpdateOperatorSpec(resourceVersion string, spec *operatorv1.OperatorSpec) (*operatorv1.OperatorSpec, string, error) {
+	return nil, "", fmt.Errorf("not implemented")
+}
+
+func (c OperatorClient) UpdateOperatorStatus(resourceVersion string, status *operatorv1.OperatorStatus) (*operatorv1.OperatorStatus, error) {
+	return nil, fmt.Errorf("not implemented")
+}
```

Comment:
we don't need this if we get https://github.com/openshift/library-go/pull/1105 merged first

Reply:
I proposed slightly different (more async) approach to this here: https://github.com/openshift/cluster-policy-controller/pull/61#issuecomment-863886456

Reply:
unfortunately that does not help, as long as the errors are returned here we're not moving forward by merging this PR

Reply:
@stlaz @s-urbaniak So it seems this PR is blocked until https://github.com/openshift/library-go/pull/1105 gets in.

Reply:
Just in case - I copy-pasted the `NoOpOperatorClient` temporarily into my PR as well.